syft/pkg/cataloger/golang/scan_binary.go

```diff
@@ -30,14 +37,42 @@ func scanFile(reader unionreader.UnionReader, filename string) ([]*debug.BuildIn
 		if bi == nil {
 			continue
 		}
-		builds = append(builds, bi)
+
+		v, err := getCryptoInformation(r)
+		if err != nil {
+			log.WithFields("file", filename, "error", err).Trace("unable to read golang version info")
+			continue
+		}
+
+		builds = append(builds, &ExtendedBuildInfo{bi, v})
 	}
 
 	archs := getArchs(readers, builds)
 
 	return builds, archs
 }
 
+func getCryptoInformation(reader io.ReaderAt) (string, error) {
```

Comment:
this seems to be derived from https://github.com/rsc/goversion/blob/3a30cee7003ec5dc0903682e79b3059c8e101952/main.go#L171-L183 however, I feel that the output is a little different can be awkward:
```
boring crypto
standard crypto
boring crypto +crypto/tls/fipsonly
standard crypto +crypto/tls/fipsonly
boring AND standard crypto!!! +crypto/tls/fipsonly
```
I think it would be more straightforward with a comma delimited list of values:
```
boring
standard
boring,crypto/tls/fipsonly
standard,crypto/tls/fipsonly
boring,standard,crypto/tls/fipsonly
```

or even better, allow that to be a native string slice:
```
["boring-crypto"]
["standard-crypto"]
["boring-crypto", "crypto/tls/fipsonly"]
["standard-crypto", "crypto/tls/fipsonly"]
["boring-crypto", "standard-crypto"]
["boring-crypto", "standard-crypto", "crypto/tls/fipsonly"]
```

This last way is more machine readable.

Reply:
I agree, will do.